src/main/java/io/github/fusionflux/portalcubed/framework/model/emissive/EmissiveWrapper.java

```diff
@@ -25,10 +26,8 @@ public BakedModel modifyModelAfterBake(BakedModel model, Context context) {
 	}
 
 	private SimpleBakedModel getSimpleBakedModel(BakedModel model) {
-		if (model instanceof SimpleBakedModel simple) {
+		if (WrapperBakedModel.unwrap(model) instanceof SimpleBakedModel simple) {
```

Comment:
Why

Reply:
H didnt mean to commit this, WrapperBakedModel.unwrap is the way you're supposed to get the actual model though